.env.development

```diff
@@ -0,0 +1,15 @@
+# Environment variables used by the app
```

Comment:
ups, these were not meant to be committed

Reply:
😬